python/tests/kat/t_tracing.py

```diff
@@ -766,11 +766,11 @@ def queries(self):
         for i in range(100):
               yield Query(self.url("target/"), phase=1)
 
-
-        # ...then ask the Zipkin for services and spans. Including debug=True in these queries
-        # is particularly helpful.
-        yield Query("http://zipkin-v1:9411/api/v2/services", phase=check_phase)
```

Comment:
Super nitty nit: the only change to that `yield Query…` line is that a single space was added to the end of the line.

1. I recommend that you configure your editor to trim trailing whitespace on save. It should already be doing that if it obeys the `.editorconfig` file, which you should totally configure your editor to do. You use VS Code, right? There's an editorconfig plugin for VS Code: https://editorconfig.org/ https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig

2. We should totally start using the `black` Python code formatter, the way we use `gofmt`, so that we don't have to have review comments like this.  There used to be many competing Python formatters, but `black` has very much been standard since 2019.

Reply:
Yeah, I agree that we should adopt something. Don't have an opinion on the formatter for python but having it declared in our developing guidelines will be good. So that the variances in editors/plugins don't add this type of noise.